frontend/src/static/js/v_zoom.js

```diff
@@ -118,12 +118,7 @@ window.vZoom = {
 
   methods: {
     initiate() {
-      if (this.info.zUser) {
-        // This code should always run since zUser must be defined
-        this.updateFileTypes();
-        this.selectedFileTypes = this.fileTypes.slice();
-      }
-
```

Comment:
Why do you remove the need for checking `zUser`? So it may crash now?

Reply:
I think the code crashes regardless if zUser is not defined since the html portion also uses it.

In some previous PR I made it that zUser should be defined the moment the v_zoom.js component is loaded. As such checking for zUser should be redundant as zUser not being defined at the moment which this piece of code runs is a bug. Hence I thought that it would perhaps be better for the code to crash earlier.

Reply:
OK